pkg/driver/wsl2/vm_windows.go

```diff
@@ -94,7 +94,7 @@ func provisionVM(ctx context.Context, instanceDir, instanceName, distroName stri
 	}
 	// path should be quoted and use \\ as separator
 	bootFileWSLPath := strconv.Quote(limaBootFileWinPath)
-	limaBootFilePathOnLinuxB, err := exec.Command(
+	limaBootFilePathOnLinuxB, err := exec.CommandContext(ctx,
```

Comment:
Since all the other arguments are on separate lines:

```suggestion
	limaBootFilePathOnLinuxB, err := exec.CommandContext(
		ctx,
```